plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueConverter.java

```diff
@@ -174,6 +174,11 @@ else if (isIcebergTable(tableParameters)) {
             partitionColumns = ImmutableList.of();
             storage = FAKE_PARQUET_STORAGE;
         }
+        else if (isDeltaLakeTable(tableParameters)) {
```

Comment:
Should similar fix be done for v1 version `GlueToTrinoConverter` ?

Reply:
I'm not sure

Reply:
It's already handled in my understanding: https://github.com/trinodb/trino/blob/e3b62bd026f3941ffdb19baee8f88aa030f4cb1a/plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/v1/converter/GlueToTrinoConverter.java#L156

Reply:
I applied the same fix for v1. I ran the test with v1 manually, but did not add it in the commit, since v1 is to be removed soon.

@pajaks thanks for spotting this!